crypto/3des/README

```diff
@@ -1,6 +1,7 @@
 How to use 3des-file-encrypt.c
 
-1)  Start by typing make
+1) a. Compile with ./configure --enable-pwdbased --enable-3des
```

Comment:
Can you clarify this is ./configure in wolfSSL directory and make, make install. Then run the Makefile in crypto/3des directory.